src/components/chat/chat-panel-header.tsx

```diff
@@ -10,19 +10,19 @@ interface IProps {
   activeNavTab: string;
```

Comment:
Can you remove the unused imports at the top of this file?

Reply:
Removed Dispatch and SetStateAction